src/human-readable/types/signatures.ts

```diff
@@ -207,6 +213,7 @@ export type SolidityKeywords =
   | 'var'
   | 'view'
   | 'virtual'
+  | AbiType
```

Comment:
Might be nice to test that this doesn't drastically slow down performance. Likely shouldn't since it's only an additional 87 members by default, but if someone is using `FixedArrayMinLength`, `FixedArrayMaxLength`, and/or `ArrayMaxDepth` it could be much more. (I wouldn't spend much time on this other than a quick sanity check.)

Reply:
Fiddling around with those configs options end ups slowing down the performance sooner than not having the `Abitype` check. Found a happy medium though. Wondering if you think it's a reasonable alternative.